Update/some_008.txt

```diff
@@ -398,24 +398,24 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6f6599>珠子</color>", NULL, "<color=#6f6599>Tamako</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 38, "ps3/s04/38/310400199", 256, TRUE);
 	OutputLine(NULL, "「ああ見えて、結構気にしてたんだよ？",
-			NULL, "\"Despite how she behaves, she really cares a lot, you know?", Line_WaitForInput);
+			NULL, "\"Despite how she behaves, she's really worried, you know?", Line_WaitForInput);
 	ModPlayVoiceLS(4, 38, "ps3/s04/38/310400200", 256, TRUE);
 	OutputLine(NULL, "　あのセンターなら変なトラブルに巻き込まれることはないはず、って夏美にアルバイトを世話したのは、千紗登だったからね」",
-			NULL, " Chisato believes you would have never been caught up in this strange incident if she hadn't introduced you to the part time job at the center.\"", Line_Normal);
+			NULL, " She said she told you about the job there because it wasn't the sort of place that should have been cause for strange incidents.\"", Line_Normal);
```

Comment:
```suggestion
			NULL, " She said she told you about the job there because she thought it wasn't the sort of place where you'd get caught up in strange incidents.\"", Line_Normal);
```